sassoftware/vscode-sas-extension
# feat: l10n infrastructure

**Summary**
Support #189 with [vscode-l10n](https://github.com/microsoft/vscode-l10n).
Added a `pseudoLocalize` script.
Note that it only applies to `client` directory. `server` is already localized with SAS tooling.

**Testing**

1. Checkout branch code
2. `npm install`, `npm run pseudoLocalize`, `vsce package`
3. Install the vsix
4. Install [Pseudo Language Language Pack](https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-qps-ploc) extension
5. Run `Configure Display Language` command and switch to the Pseudo Language. (It requires restart)
6. See UI text provided by the extension are pseudo translated.
